CMakeLists.txt

```diff
@@ -137,6 +137,7 @@ option(CXXTEST_ADD_PERFORMANCE
 add_subdirectory(Framework)
 
 include_directories(Framework/Kernel/inc)
+include_directories(Framework/Json/inc)
```

Comment:
As Kernel requires this can be swap the order here to make that slightly clearer?